src/reducers/systemIntakesReducer.ts

```diff
@@ -1,19 +1,45 @@
-import { STORE_SYSTEM_INTAKES } from '../constants/actions';
-import { SystemIntakesState } from '../types/systemIntake';
+import { SystemIntakesState } from 'types/systemIntake';
+import { prepareSystemIntakeForApp } from 'data/systemIntake';
+import { fetchSystemIntakes } from 'routines/routines';
+import { DateTime } from 'luxon';
 
 const initialState: SystemIntakesState = {
-  systemIntakes: []
+  systemIntakes: [],
+  loaded: false,
+  loadedTimestamp: null,
+  error: null
 };
 
 function systemIntakesReducer(
   state = initialState,
   action: any
 ): SystemIntakesState {
   switch (action.type) {
-    case STORE_SYSTEM_INTAKES:
+    case fetchSystemIntakes.TRIGGER:
       return {
         ...state,
-        systemIntakes: action.systemIntakes
+        loaded: false
+      };
+    case fetchSystemIntakes.REQUEST:
+      return state;
+    case fetchSystemIntakes.SUCCESS:
+      console.log(action);
+      return {
+        ...state,
+        systemIntakes: action.payload.map((intake: any) =>
+          prepareSystemIntakeForApp(intake)
+        )
+      };
+    case fetchSystemIntakes.FAILURE:
+      return {
+        ...state,
+        error: action.payload
+      };
+    case fetchSystemIntakes.FULFILL:
```

Comment:
`FULFILL` happens after every routine, regardless of `SUCCESS` or `FAILURE`. `IsLoaded` shouldn't be set `true` here because if there's a `FAILURE` data wasn't loaded.

However, if we use `isLoading` (present tense), it can be set to `false` in here because it is set to false regardless of `SUCCESS` or `FAILURE`

Reply:
The reason I went with `isLoaded` over `isLoading` is because I wanted to differentiate when the intakes are loaded, rather than when the routine is not yet begun.

Reply:
We could also make this an enum? Of `LOAD_UNSTARTED` `LOAD_IN_PROGRESS` `LOAD_SUCCESS` `LOAD_FAILURE`

We'd probably reuse such a type, especially if we starting doing more robust error handling